templates/AureliaSpa/Views/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>@ViewData["Title"] - Aurelia</title>
+        <base href="/" />
+    </head>
+    <body aurelia-app="boot">
```

Comment:
Even if this is needed, could you move it into `Views/Home/Index.cshtml`, since it's only that one view that really needs to start up the Aurelia app?